python/ray/workers/default_worker.py

```diff
@@ -124,6 +124,11 @@
     type=int,
     default=0,
     help="The PID of the process for setup worker runtime env.")
+parser.add_argument(
+    "--startup-token",
+    required=True,
+    type=int,
+    help="The startup token of the process for setup worker runtime env.")
```

Comment:
This should be `"The startup token assigned to this worker process by the raylet"` (the startup token has nothing to do with the runtime env).

Reply:
Ah! I will fix the docs in Python code.